src/main/java/net/snowflake/client/core/SFBaseSession.java

```diff
@@ -332,7 +332,7 @@ public HttpClientSettingsKey getHttpClientKey() throws SnowflakeSQLException {
 
       return ocspAndProxyKey;
     }
-    // If JVM proxy parameters are specified, https proxies need to go through the JDBC driver's
```

Comment:
JVM proxy properties for both http and https proxies are now passed in to the JDBC driver (but can be overridden still by connection proxy properties). This allows them to be used for PUT/GET.